src/orchestration/Orchestration.ts

```diff
@@ -63,6 +63,7 @@ export type PartialConfig = {
     batchLimit?: number;
     clientBuilder?: ClientBuilder;
     cookieAttributes?: PartialCookieAttributes;
+    customAttributesMap?: { [k: string]: string | number | boolean };
```

Comment:
todo: We should brainstorm some names for this . E.g.,
- `sessionAttributes`

Reply:
Updated to use `sessionAttributes`